Description: Add unistd.h avoid build errors. 
 Function in unistd.h not found on aarch64
 architecture. Add unistd.h to help find functions.
Forwarded: not-needed
Last-Update: 2023-10-23
---

 src/worker/aptlock.cpp   |    2 ++
 src/worker/aptworker.cpp |    1 +
 2 files changed, 3 insertions(+)

--- a/src/worker/aptlock.cpp
+++ b/src/worker/aptlock.cpp
@@ -23,6 +23,8 @@
 #include <apt-pkg/error.h>
 #include <QDebug>
 
+#include <unistd.h>
+
 AptLock::AptLock(const QString &path)
     : m_path(path.toUtf8())
     , m_fd(-1)
--- a/src/worker/aptworker.cpp
+++ b/src/worker/aptworker.cpp
@@ -43,6 +43,7 @@
 #include <apt-pkg/upgrade.h>
 #include <apt-pkg/versionmatch.h>
 #include <string>
+#include <unistd.h>
 
 // System includes
 #include <sys/statvfs.h>
